next/package.json

```
@@ -4,12 +4,10 @@
   "private": true,
   "dependencies": {
     "@prisma/client": "^5.21.1",
-    "@prisma/instrumentation": "^5.21.1",
     "@reduxjs/toolkit": "^2.2.8",
-    "@sentry/core": "^9.10.1",
-    "@sentry/nextjs": "^9.10.1",
-    "@sentry/profiling-node": "^9.10.1",
-    "@sentry/utils": "^8.36.0-beta.0",
+    "@sentry/core": "^10.8.0",
+    "@sentry/nextjs": "^10.8.0",
+    "@sentry/profiling-node": "^10.9.0",
```

```suggestion
    "@sentry/core": "^10.17.0",
    "@sentry/nextjs": "^10.17.0",
    "@sentry/profiling-node": "^10.17.0",
```

Generally we should not depend on core, do you know why we need this dependency?